### PR TITLE
Cherry men 5547 3.0.x

### PR DESCRIPTION
**Pull request by oleorhagen**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline --pr mender-artifact/415

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ PKGS = $(shell go list ./... | grep -v vendor)
 PKGFILES = $(shell find . \( -path ./vendor -o -path ./Godeps \) -prune \
 		-o -type f -name '*.go' -print)
 PKGFILES_notest = $(shell echo $(PKGFILES) | tr ' ' '\n' | grep -v '\(client/test\|_test.go\)' )
-GOCYCLO ?= 15
+GOCYCLO ?= 20
```
Comment (kacf) [inline Makefile:16]: Why only on 3.0.x?

**Review by kacf — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (oleorhagen) [reply Makefile:16]: Tbh, I just assumed we'd bumped it and `3.0` was not, but I see I was wrong, I'll have another look, it's a little odd

Comment (kacf) [reply Makefile]: Yes, absolutely.

For master I smell a QA ticket. 🐕

Comment (oleorhagen) [reply Makefile]: It seems `golangci-lint` does cyclomatic complexity checking ,but it's set at `20`.

This means that the old `Makefile` GoCyclo` check is not really relelevant anylonger.

I'd vote for removing it. @lluiscampos what do you think?

Comment (oleorhagen) [reply Makefile:16]: Discussed f2f:
- Bumping to 20 for now. @lluiscampos follows up with a ticket for `master` to fix this for `golangci-lint`

**Pull request merged**